src/pages/AddDrinkPage/AddDrinkPage.styled.js

```diff
@@ -0,0 +1,358 @@
+    &::before {
+      content: "";
+      background-color: white;
+      border: 2px solid var(--transp-white);
+    }
+    &::after {
+      content: "";
+      display: flex;
+      border-radius: 50%;
+      top: 2.2px;
+      left: 2.2px;
+      border: 3.1px solid black;
+      width: 12px;
+      height: 12px;
+      opacity: 1;
+      position: absolute;
+      z-index: 3;
+    }
+  }
+  position: relative;
+
+  input[type="radio"] {
+    opacity: 0;
+  }
+  input[type="radio"] + p {
+    padding-left: 15px;
+    &::before {
+      content: "";
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border-radius: 50%;
+      border: unset;
+      border: 3px solid var(--transp-white);
+      width: 17px;
+      height: 17px;
+      opacity: 1;
+      position: absolute;
+      z-index: 2;
+      top: 0;
+      left: 0;
+    }
+  }
+`;
+export const StyledRadioLabelDiv = styled.label`
+  display: flex;
+  gap: 30px;
+  @media ${device.tablet} {
+    padding-top: 30px;
+    max-height: 32px;
+  }
+`;
+
+export const StyledIngTitle = styled.p`
+  font-size: 28px;
+  font-weight: 600;
+  line-height: 32px;
+`;
+export const StyledIngTitleDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const StyledAddDiv = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 104px;
+  border-radius: 200px;
+  border: 1px solid var(--transp-white);
+  height: 38px;
+  margin-bottom: 34px;
+`;
+export const StyledAddBtn = styled.button`
+  background-color: transparent;
+  border: none;
+  width: 16px;
+  height: 16px;
+  font-size: 30px;
+  line-height: 0;
+  padding: 0;
+  justify-content: center;
+  align-items: center;
+  color: inherit;
+  font-family: inherit;
+  &:disabled {
+    color: var(--transp-white);
+  }
+`;
+export const StyledIngFieldInput = styled.input`
+  width: 101px;
+  height: 50px;
+  border-radius: 200px;
+  border: 1px solid var(--transp-white);
+  background: transparent;
+  color: var(--white);
+  padding: 0 60px 0 18px;
+  position: relative;
+  &::after {
+    content: "cl";
+    display: block;
+    width: 10px;
+    height: 10px;
+    color: white;
+    right: 30px;
+    top: 10px;
+  }
+  @media ${device.tablet} {
+    width: 150px;
+  }
+`;
+export const StyledIngFieldLabel = styled.label`
+  display: flex;
+  margin-bottom: 18px;
+  gap: 8px;
+`;
+export const StyledIngFieldWrapper = styled.div`
+  position: relative;
+`;
+export const StyledAfterContent = styled.span`
+  content: "cl";
+  display: block;
+  width: 10px;
+  height: 10px;
+  color: white;
+  position: absolute;
+  right: 50px;
+  top: 20px;
+  transform: translateY(-50%);
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 18px;
+`;
+export const StyledIngFieldBtn = styled.button`
+  background: transparent;
+  border: none;
+  padding: 0;
+  @media ${device.tablet} {
+    margin-left: 188px;
+  }
+  @media ${device.laptopL} {
```

Comment:
масив девайсів було змінено
```suggestion
  @media ${device.desktop} {
```